js/components/discussions/threads.vue

```diff
@@ -166,17 +192,7 @@ export default {
         }
     },
     ready() {
-        this.$api.get('discussions/', {for: this.subjectId}).then(response => {
-
-            this.loading = false;
-            this.discussions = response.data;
-
-            if (document.location.hash) {
-                this.$nextTick(() => { // Wait for data to be binded
-                    this.jumpToHash(document.location.hash);
-                });
-            }
-        }).catch(log.error.bind(log));
+        this.discussions.fetch({for: this.subjectId});
```

Comment:
I _think_ using this instead of `$api.get` makes us dependent on `swagger.json` on the frontend, which we avoided until now and clearly don't want. Can you check?

Reply:
:skull:

Reply:
😇